src/runtime_src/core/tools/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ if (${XRT_NATIVE_BUILD} STREQUAL "yes")
 endif()
 
 install (PROGRAMS "./common/xball" DESTINATION ${XRT_INSTALL_BIN_DIR})
+install (FILES "./common/xball.py" DESTINATION ${XRT_INSTALL_PYTHON_DIR})
```

Comment:
in non-native build (edge build) , XRT_INSTALL_PYTHON_DIR is not defined. I think xball is not requried in edge case as there will be only one device 
You can move these under NATIVE_BUILD condition

Reply:
That makes a lot of sense, I was wondering why it was failing on the pipeline! Thank you!